stretch_core/stretch_core/joint_trajectory_server.py

```diff
@@ -247,7 +256,7 @@ def execute_cb(self, goal_handle):
                 return self.error_callback(goal_handle, FollowJointTrajectory.Result.INVALID_JOINTS, 'no trajectory in goal contains enough waypoints')
             n_points = max([len(trajectory.points) for trajectory in trajectories])
             duration = max([Duration.from_msg(trajectory.points[-1].time_from_start) for trajectory in trajectories])
-            self.node.get_logger().info(f"{self.node.node_name} joint_traj action: new traj with {n_points} points over {to_sec(duration.to_msg())} seconds")
+            self.node.get_logger().info(f"{self.node.node_name} joint_traj action: new traj with {n_points} points over {round(duration.nanoseconds()/1e9, 2)} seconds")
```

Comment:
This change raises a "TypeError: 'int' object is not callable" error for me when I try to keyboard teleop in trajectory mode as it tries to call duration.nanoseconds which is an int object. Is this necessary?

Reply:
You're right, looks like the change didn't make it to the PR. It is now added in [`4113647`](https://github.com/hello-robot/stretch_ros2/commit/4113647eb8bf1887928aa805595b5a0e351e796d)